Gemfile

```diff
@@ -27,6 +27,9 @@ gem 'stimulus-rails'
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'jbuilder'
 
+gem 'ffi'
+gem 'rails-controller-testing'
+gem 'rspec-rails', '~> 6.0.0.rc1'
```

Comment:
- [optional] Hi @JuanLPalacios, this gem needs to be in the development and test group. This exposes generators and rake tasks without having to type RAILS_ENV=test.